util/libcrypto.num

```diff
@@ -4533,3 +4533,13 @@ DH_get0_q                               4474	1_1_0i	EXIST::FUNCTION:DH
 DH_get0_g                               4475	1_1_0i	EXIST::FUNCTION:DH
 DH_get0_priv_key                        4476	1_1_0i	EXIST::FUNCTION:DH
 DH_get0_pub_key                         4477	1_1_0i	EXIST::FUNCTION:DH
+RSA_get0_p                              4478	1_1_1	EXIST::FUNCTION:RSA
```

Comment:
@richsalz, @levitte,  I just noticed that I marked the  DH getter symbols with `1_1_0i`, because I originally intended to backport to 1.1.0. What do you prefer?
* backport simple DH,DSA,RSA getters to 1.1.0
* replace `1_1_0i` by `1_1_1` for the DH getters

Reply:
IMHO the accessors should be backported to 1.1.0 to be really useful - so we do not have such basic API divergence between 1.1.1 and 1.1.0.

Reply:
This does touch a grey area...  by policy, new functions should come in new feature releases only, but we have made an exception for obviously *missing* constructors, destructors, setters and getters.  However, this PR isn't about *missing* getters, it's about the available getters being awkward to some, i.e. these added getters are a convenience.

What say the rest of @openssl?

Reply:
Since 1.1.1 will be LTS and 1.1.0 will not, one could argue that it‘s not worth backporting.

Reply:
That too... but then, please make sure to fix the version number in libcrypto.num

Reply:
I agree with the LTS vs not and am slightly against backporting, but it's not something I'd really argue about if things went the other way.